.gitignore

```diff
@@ -604,3 +605,7 @@ Makefile.am-stamp
 
 # /src/mpid/ch4/include
 /src/mpid/ch4/include/netmodpre.h
+
+# /src/mpid/ch4/shm/posix
+src/mpid/ch4/shm/posix/eager/include/posix_eager_pre.h
+src/mpid/ch4/shm/posix/posix_eager_array.c
```

Comment:
Sorry I missed these. I almost never build in-tree. Should these be in `src/mpid/ch4/.gitignore`?

Reply:
Most likely they should. In that case, we should move the remaining ch4 related files in the top-level `.gitignore` as well.